consul-core/pom.xml

```diff
@@ -31,5 +31,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
```

Comment:
instead of hamcrest, can you use the assertj matchers that come with Dropwizard already?